src/components/creator/WorkoutCreator.js

```diff
@@ -0,0 +1 @@
+import header from '../notglobal/header'
```

Comment:
This is importing something that doesn't exist?

Reply:
Oh yea we were just running over an example of something forgot to delete that.